viite-UI/src/less/core/forms.less

```diff
@@ -290,7 +290,17 @@ form.roadAddressProject .control-label-small {
   width: 55px !important;
   text-align: center;
 }
+form.roadAddressProject .junction-control-label-small{
```

Comment:
This form is to display junction points - shouldn't be linked to the roadAddressProject class.

Reply:
Fixed with new commit